src/appleseed-max-impl/appleseedoslplugin/oslshaderregistry.cpp

```diff
@@ -426,8 +428,9 @@ void OSLShaderRegistry::create_class_descriptors()
             L"oslTextureMapParams",                     // internal parameter block's name
             0,                                          // ID of the localized name string
             class_descr,                                // class descriptor
-            P_AUTO_CONSTRUCT,                           // block flags
-
+            P_AUTO_CONSTRUCT | P_VERSION,               // block flags
+                                                        // --- P_VERSION arguments ---
+            1,
```

Comment:
That's supposed to be on the previous line no?

Reply:
Probably, since it's only one argument...